qipr_approver/approver/views/project.py

```diff
@@ -26,7 +26,8 @@ def project(request, project_id=None):
         title = project_form.get('title')
         project = project_crud.get_project_or_none(project_id)
         if(project is None):
-           project = project_crud.create_or_update_project(current_user, project_form, project_id)
+            project = project_crud.create_or_update_project(current_user, project_form, project_id)
+            return redirect(reverse("approver:similar_projects", args=[str(project.id)]))
```

Comment:
@PFWhite , do we want this to only check similar projects upon first creation? What if the customer did a quick rough draft of the project before filling in the details later?